backend/src/routes/api/storage-class/utils.ts

```diff
@@ -0,0 +1,58 @@
+import { FastifyRequest } from 'fastify';
+import { K8sResourceCommon, KubeFastifyInstance, StorageClassConfig } from '../../../types';
+import { isHttpError } from '../../../utils';
+import { errorHandler } from '../../../utils';
+
+export async function updateStorageClassMetadata(
```

Comment:
Just to reiterate Juntao's comment, I think this is meant to be updated to be more specific: `updateStorageClassConfigAnnotation` or without the `Annotation`.